main.py

```diff
@@ -1,7 +1,13 @@
 # -*- coding: utf-8 -*-
-
 from __future__ import print_function
 
+from numpy.random import seed
+seed(1) # this is a test, not using RND_SEED variable here
```

Comment:
Remove the comment, as it looks like it's a "note to self" type comment made during development.

Reply:
Done: https://github.com/RSE-Sheffield/nmt-keras/commit/3909db582df38dfbc30c7158485ac5333538099e